### DISCUSSION TITLE
Highlight on not only keywords

### DISCUSSION THREAD

**Discussion by ivanlauhk** [night/betterttv#6786] [Ideas]
Currently, the highlighting system depends on keywords to trigger highlights. However, there could be more options for users to choose:

- Reply message (Right now a reply where the parent message nor the reply itself doesn't contain a highlight wouldn't trigger the highlight to occur)
- First time chat
- Raider message
- Mods
- VIP

**Comment by ZaCloud** [comment]
YES!
And adding onto this: Usernames. I want to know if certain users say something, but the way it is now, it only highlights if someone else mentions them, even if I use the @username format.

Useful for:

- if a friend (or someone I'm a fan of) joins a busy chat & says stuff.

- if I want to watch how a situation develops, like if a certain user is having a hard time in their life, & are occasionally updating with how they're doing, if things are getting better for them or not, etc.

- just plain keeping track of interesting conversations that don't involve me.

- the streamer's spouse/significant other/friend says something in chat that the streamer might have missed, or they're joking around with chat & I don't want to miss the comedy.

**Comment by night** [comment]
Highlight keywords now match against the parent message of a reply, so replies to messages containing your keywords highlight as expected. Closing as resolved.
